src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTestBase.cs

```diff
@@ -287,6 +287,7 @@ internal static string GetMapActionString(string sources, string className = "Te
 using Microsoft.AspNetCore.Http.Generators.Tests;
 using Microsoft.Extensions.Primitives;
 using Microsoft.Extensions.DependencyInjection;
+using Http;
```

Comment:
```suggestion
```

I think we can remove this?

Reply:
If I remove that line, the new tests fail to compile.

The test is whether the request delegate generator generates compilable code when a service from the namespace "Http" is explicitly requested from DI. There doesn't seem to be a way to add to that list of imports per-test, so I had to add it to the universal template.

I could alternatively `global`ly qualify the type in the source string in `RequestDelegateGeneratesCompilableCodeForServiceInNamespaceHttp` and `RequestDelegateGeneratesCompilableCodeForKeyedServiceInNamespaceHttp`, but that felt worse to me. I'll do it at your advisement, of course.

Reply:
Ah! 🤦🏽‍♀️ I failed to realize that this is the imports for the test code, not the imports for the file. All good here!